bgcval2/download_from_mass.py

```diff
@@ -418,59 +423,75 @@ def downloadMass(jobID, doMoo=True):
+    #print "running the command:",bashCommand
+    #process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    #output = process.communicate()[0]
+    #print "output",output
 
-        #bashCommand = "moo passwd -r"
-        #print "running the command:",bashCommand
-        #process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
-        #output = process.communicate()[0]
-        #print "output",output
+    bashCommand = "moo ls moose:/crum/" + jobID + "/ony.nc.file/*.nc"
+    download_script_txt = ''.join([download_script_txt, bashCommand, '\n'])
 
-        bashCommand = "moo ls moose:/crum/" + jobID + "/ony.nc.file/*.nc"
-        print("running the command:", bashCommand)
+    if not doMoo:
+        print("download_from_mass:\tthe command is (dry-run): \n", bashCommand)
+        output = ''
+    else:
+        print("download_from_mass:\trunning the command:", bashCommand)
         stdout.flush()
-
         process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
         output = process.communicate()[0]
 
-        if len(output.split('\n')) > 6000:
-            #bashCommand = ''
-            failed = 0
-            process1 = {}
-            output1 = {}
-            #for i in reversed(range(100)):
-            for i in range(100):
-
-                bashCommand = "moo get --fill-gaps moose:/crum/" + jobID + "/ony.nc.file/*_1y_??" + mnStr(
-                    i) + "*.nc " + outputFold
-                print("running the command:", bashCommand)
-                stdout.flush()
-                try:
-                    process1[i] = subprocess.Popen(bashCommand.split(),
-                                                   stdout=subprocess.PIPE)
-                    process1[i].wait()
-                    output1[i] = process.communicate()[0]
-                    print(i, output1[i])
-                except:
-                    failed += 1
-                    print("Failed", i, '\t', bashCommand)
-            if failed > 10:
-                assert 0
+    if len(output.split('\n')) > 6000:
+        failed = 0
+        process1 = {}
+        output1 = {}
+        for i in range(100):
+            bashCommand = "moo get --fill-gaps moose:/crum/" + jobID + "/ony.nc.file/*_1y_??" + mnStr(
+                i) + "*.nc " + outputFold
+            print("download_from_mass:\trunning the command:", bashCommand)
+            download_script_txt = ''.join([download_script_txt, bashCommand, '\n'])
+
+            stdout.flush()
+            try:
+                process1[i] = subprocess.Popen(bashCommand.split(),
+                                               stdout=subprocess.PIPE)
+                process1[i].wait()
+                output1[i] = process.communicate()[0]
+                print("download_from_mass:\t",i, output1[i])
+            except:
+                failed += 1
+                print("Failed", i, '\t', bashCommand)
+        if failed > 10:
+            assert 0
```

Comment:
this really has no effect - write a message, throw a warning, or exit completely maybe?